src/inverse-client.ts

```diff
@@ -86,40 +63,31 @@ export class RestClient {
     return this.requestWrapper.get('v2/public/trading-records', params);
   }
 
-  getSymbols(): GenericAPIResponse {
-    return this.requestWrapper.get('v2/public/symbols');
-  }
-
-  /**
-   * @deprecated use getLiquidations() instead
-   */
-  getPublicLiquidations(params: {
+  getMarkPriceKline(params: {
     symbol: string;
-    from?: number;
+    interval: string;
+    from: number;
     limit?: number;
-    start_time?: number;
-    end_time?: number;
   }): GenericAPIResponse {
-    return this.getLiquidations(params);
+    return this.requestWrapper.get('v2/public/mark-price-kline', params);
   }
-
-  getLiquidations(params: {
+  
+  getIndexPriceKline(params: {
     symbol: string;
-    from?: number;
+    interval: string;
+    from: number;
     limit?: number;
-    start_time?: number;
-    end_time?: number;
   }): GenericAPIResponse {
-    return this.requestWrapper.get('v2/public/liq-records', params);
+    return this.requestWrapper.get('v2/public/index-price-kline', params);
   }
-
-  getMarkPriceKline(params: {
+  
+   getPremiumIndexKline(params: {
```

Comment:
```suggestion
  getPremiumIndexKline(params: {
```

Extra space can be removed